draft/2022-02-09-this-week-in-rust.md

```diff
@@ -19,14 +19,20 @@ If you find any errors in this week's issue, [please submit a PR](https://github
 ### Foundation
 
 ### Project/Tooling Updates
+* [Nushell v0.44](https://www.nushell.sh/blog/2022-02-08-nushell_0_44.html)
+* [Rust API for the Firefox Profiler](https://blog.mozilla.org/performance/2022/02/08/rust-api-for-the-firefox-profiler/)
+* [Starship v1.3.0](https://github.com/starship/starship/releases/tag/v1.3.0)
```

Comment:
While this update certainly looks very cool, I don't feel there is enough content beyond the list of changes to feature it in the newsletter (I'm looking for more content about how to use the project, how it helps someone do cool things with Rust, etc.) My fellow editors and I are working on some better guidelines for project updates and will be publishing them within the next few weeks. I will take this link out before I merge this PR.

Reply:
Good idea. Thanks!